applications/portal/frontend/src/components/Submit/CommercialForm.tsx

```diff
@@ -19,7 +19,7 @@ import { AlertIcon } from "../icons";
 import StepNavigation from "./StepNavigation";
 import { useFormik } from "formik";
 import * as yup from "yup";
-import { addAntibody } from "../../services/AntibodiesService";
+import { postNewAntibody } from "../../utils/antibody";
```

Comment:
why the logic is for the post has been moved from the service to utils?

Reply:
utils should be pure functions

Reply:
moved to helpers